test/jobs/kps_address_job_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class KpsAddressJobTest < ActiveJob::TestCase
+  %w[
+    KpsAddressCreateJob
+    KpsAddressUpdateJob
+  ].each do |property|
+    test "can enqueue #{property}" do
+      assert_enqueued_jobs 0
+      property.constantize.perform_later(addresses(:formal))
+      assert_enqueued_jobs 1
+    end
+
+    test "can initialize KPS queries as #{property} for given user and address" do
+      client = Minitest::Mock.new
+      def client.sorgula
+        true
+      end
+
+      Services::Kps::Omu::Adres.stub :new, client do
+        assert property.constantize.perform_later({}) # dummy hash
+      end
+    end
+  end
+
+  # Skip this block on CircleCI since it needs IP permissions to run.
+  unless ENV['CI']
```

Comment:
disclaimer: "keyfe keder"

bu unless'i kaldırıp comment'i mesaj kılarak test bloğu içinde ilk satırda şöyle de yapabiliyor olman lazım (?):

```ruby
skip 'Skipping on CircleCI since it needs IP permissions to run'
```